vcpkg/get_started/get-started-vs.md

```diff
@@ -0,0 +1,150 @@
+    * Choose "CMake Project" as the template.
+    * Name your project "helloworld"
+    * Check the box for "Place solution and project in the same directory."
+    * Click the "Create"
+
+2. Configure the `VCPKG_ROOT` environment variable.
+   
+    Open the built-in Developer PowerShell window (``Ctrl+` ``) in Visual Studio and run the following commands:
+
+    ```console
+    $env:VCPKG_ROOT = "C:\path\to\vcpkg"
+    $env:PATH = "$env:VCPKG_ROOT;$env:PATH"
+    ```
+
+    Setting `VCPKG_ROOT` helps Visual Studio locate where your vcpkg instance is located.
+    Adding it to `PATH` ensures you can run vcpkg commands directly from the shell.
+
+3. Generate a Manifest file and add dependencies.
+
+    Run the following command to create a vcpkg manifest file (`vcpkg.json`):
+
+    ```console
+    vcpkg new --application
+    ```
+
+    The command adds a `vcpkg.json` and a `vcpkg-configuration.json` file to your project directory. 
+
+    Add the `fmt` package as a dependency:
+
+    ```console
+    vcpkg add port fmt
+    ```
+
+    Your `vcpkg.json` should now contain:
+
+    ```json
+    {
+        "dependencies": [
+            "fmt"
+        ]
+    }
+    ```
+
+    This is your manifest file. vcpkg reads the manifest file to learn what dependencies to install and integrates with CMake to provide the dependencies required by your project.
+
+    The default `vcpkg-configuration.json` file introduces [baseline](../reference/vcpkg-configuration-json.md#registry-baseline) constraints, specifying the minimum versions of dependencies that your project should use. While modifying this file is beyond the scope of this tutorial, it plays a crucial role in defining version constraints for your project's dependencies. Therefore, even though it's not strictly necessary for this tutorial, it's a good practice to add `vcpkg-configuration.json` to your source control to ensure version consistency across different development environments.
+
+## 3 - Set up the project files
+
+1. Modify the `helloworld.cpp` file.
+
+    Replace the content of `helloworld.cpp` with the following code:
+
+    :::code language="cpp" source="../examples/snippets/get-started/main.cpp":::
+
+    In this `helloworld.cpp` file, the `<fmt/core.h>` header is included for using the `fmt` library. The `main()` function then calls `fmt::print()` to output the "Hello World!" message to the console.
+
+2. Configure the `CMakePresets.json` file.
+   
+   * Rename the `CMakePresets.json` to `CMakeUserPresets.json`
+   * Update its contents as shown below. Replace `<VCPKG_ROOT>` with the path to your vcpkg directory.
+  
+  ```json
+  {
+    "version": 3,
+    "configurePresets": [
+      {
+        "name": "default",
+        "cacheVariables": {
+          "CMAKE_TOOLCHAIN_FILE": "<VCPKG_ROOT>/scripts/buildsystems/vcpkg.cmake"
+        }
+      }
+    ]
+  }
+  ```
+
+  * Add `CMakeUserPresets.json` to you `.gitignore` file if you're using version control. 
```

Comment:
```suggestion
  * Since this file includes a hardcoded absolute path, it is recommended that you don't keep this file under source control. If you're using Git, add `CMakeUserPresets.json` to your `.gitignore` file.
```